### PR TITLE
produce world map of AA countries

### PR DESCRIPTION
**Pull request by joseepoirier**
Manually created the entry for Malawi to simplify the graph. We'll need to adapt the code when we have > 1 dual shock context. 
I'd vote for continuing to distinguish between dry spells and drought, or we combine them and label the category drought/dry spells since they are distinct shocks.

### CODE REVIEW

File: utils_general/aa-countries.R [hunk 1]
```diff
@@ -0,0 +1,75 @@
+###
```
Comment (Tinkaa) [inline utils_general/aa-countries.R:1]: Not regarding this line, but the directory structure. I would propose to not place this in utils_general as this are really pipeline scripts. Instead I would propose to make a `analyses/glb` folder, and place it there. What do you think?

File: utils_general/aa-countries.R [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+# Install packages
+###
+
+library(tidyverse)
+#library(rvest)
+#library(magrittr)
+#library(ggmap)
+#library(stringr)
```
Comment (Tinkaa) [inline utils_general/aa-countries.R:9]: remove unused libraries

File: utils_general/aa-countries.R [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+world_map <- map_data("world")
+
+###
+# identify AA pilots
+###
+
+# list countries and shocks
+drought <- c("Somalia", "Ethiopia", "Niger", "Burkina Faso")
+#dry_spells <- c("Malawi")
+#floods <- c("Bangladesh", "Nepal", "Malawi", "Chad", "South Sudan")
```
Comment (Tinkaa) [inline utils_general/aa-countries.R:24]: remove commented lines

File: utils_general/aa-countries.R [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+       ,subtitle = "Active and under development frameworks") +
+  theme(legend.position="bottom"
+        #text = element_text(family = "Gill Sans", color = "#FFFFFF")
+        ,plot.title = element_text(size = 24)
+        ,plot.subtitle = element_text(size = 12)
+        ,axis.text = element_blank()
+        ,axis.title = element_blank()
+        ,axis.ticks = element_blank()
+        ,panel.background = element_rect(fill = "white")
+        #,plot.background = element_rect(fill = "#444444")
```
Comment (Tinkaa) [inline utils_general/aa-countries.R:72]: here and above, remove commented lines

File: utils_general/aa-countries.R [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        ,panel.grid = element_blank()
+  )
+
```
Comment (Tinkaa) [inline utils_general/aa-countries.R:75]: don't see code to save the png? (do comment out that line but good to keep it somewhere)

**Review by Tinkaa — COMMENTED** [review]
Beautiful how ggplot can so easily create nice plots, and you nicely documented the code! 

Few things
- as commented would place the files in a different directory
- I am not a big fan of putting the pngs in the repo. Would it be alright to put it on the GDrive (`processed/glb/plots` for example) or is there a specific reason you saved it here? 
- I think it would be really nice to add the map to the repo main readme. What do you say?

File: utils_general/aa-countries.R (hunk 1 above)
Comment (joseepoirier) [reply utils_general/aa-countries.R:1]: Oh I hadn't noticed glb. Much better location for it.

File: utils_general/aa-countries.R (hunk 2 above)
Comment (joseepoirier) [reply utils_general/aa-countries.R:9]: Ha! forgot to do that. Thanks!

File: utils_general/aa-countries.R (hunk 1 above)
Comment (joseepoirier) [reply utils_general/aa-countries.R:1]: I need to figure out how to move the Rproj file with it..

File: utils_general/aa-countries.R (hunk 3 above)
Comment (joseepoirier) [reply utils_general/aa-countries.R:24]: I was leaving them in so we can easily update it / change how we represent MWI. I guess if the overall structure & pattern are clear I can remove them.

File: utils_general/aa-countries.R (hunk 1 above)
Comment (Tinkaa) [reply utils_general/aa-countries.R:1]: `glb` doesn't exist yet😁 but think this is a perfect occasion to create it

File: utils_general/aa-countries.R (hunk 3 above)
Comment (Tinkaa) [reply utils_general/aa-countries.R:24]: to me the overall structure is clear

**Comment by joseepoirier** [comment]
@Tinkaa I addressed your helpful comments, thanks. Can you take a quick look so we can merge?

**Comment by Tinkaa** [comment]
@joseepoirier Looks great! The only thing is that when I open the png now saved in `processed/glb/plots`, it doesn't look good, it is very stretched. So maybe change how you save the png? But regardless of that, ready to be merged imo.

**Comment by Tinkaa** [comment]
Aah and the png seems to still be saved on the github, would you agree to remove it from there?

File: utils_general/aa-countries.R (hunk 5 above)
Comment (joseepoirier) [reply utils_general/aa-countries.R:70]: The saving function distorts the image when it saves it. I've already spent a few hours trying to figure it out. Would say to leave it as, keeping the manually saved png for now, and I'll look into the saving issue later.

**Comment by joseepoirier** [comment]
> Aah and the png seems to still be saved on the github, would you agree to remove it from there?

Ugh that's me forgetting to actually removing it after realising the saving function issue. Sorry - I deleted it and also moved the R file + Rproj to `analyses/glb`

**Pull request merged**